src/content/docs/en/guides/internationalization.mdx

```diff
@@ -0,0 +1,268 @@
+---
+title: Internationalization
+description: Learn how to use i18n routing in Astro.
+i18nReady: false
+---
+
+import FileTree from '~/components/FileTree.astro'
+import Since from '~/components/Since.astro'
+
+Astro's internationalization (i18n) features allow you to adapt your project for an international audience.
+
+
+## i18n Routing (Experimental)
+
+<Since v="3.5.0" />
+
+Astro's experimental i18n routing allows you to add your multilingual content with support for configuring a default language and URL path convention, computing relative page URLs, and accepting preferred languages provided by your visitor's browser.
+
+This routing API helps you generate, use, and verify the URLs that your multi-language site produces. Check back and update regularly for the latest changes as this API continues to develop!
+
+
+### Configure i18n routing
+
+1. Enable the experimental routing option by adding an `i18n` object to your Astro configuration with a [default location (`defaultLocale`) and a list of all languages to support (`locales`)](#defaultlocale-and-locales):
+
+   ```js title="astro.config.mjs"
+   import { defineConfig } from "astro/config"
+   export default defineConfig({
+     experimental: {
+       i18n: {
+         defaultLocale: "en",
+         locales: ["en", "es", "pt-br"]
+       }
+     }
+   })
+   ```
+  
+    You can also specify a [`routingStrategy`](#routing-strategy) to customize the URLs for your default language.
+
+
+2. Organize your content folders with localized content by language, including your `defaultLocale`. Your folder names must match the items in `locales` exactly. `src/pages/index.astro` will now automatically redirect to the `index.astro` file of your default language.
+    <FileTree>
+    - src
+      - pages
+        - en
+          - about.astro
+          - index.astro
+        - es
+          - about.astro
+          - index.astro
+        - pt-br
+          - about.astro
+          - index.astro
+        - index.astro  (redirects to the route created for `/en/index.astro`)
+    </FileTree>
+
+    :::note
+    The localized folders do not need to be at the root of the `/pages/` folder.
+    
+    Create individual `/[locale]/` folders anywhere within `src/pages/` and Astro's [file-based routing](/en/core-concepts/routing/) will create your pages at corresponding URL paths. 
+    
+    ```
+    src/pages/blog/[locale]/about.astro -> example.com/blog/es/about/
+    ```
+    :::
+
+3. When writing links to pages on your site, use a localized URL. You can write this URL directly, or compute it using the [`getRelativeLocaleURL()`](#getrelativelocaleurl) helper available from the [`astro:i18n` module](#virtual-module-astroi18n). This will always provide the correct, localized route and can help you correctly use, or check, URLs on your site.
```

Comment:
> When writing links to pages on your site, use a localized URL. 

@ematipico Just checking on a case like:

- `prefix-other-locales` so that a default locale URL route is `example.com/blog/`
- you want to hardcode (not compute) the href and write it by hand
- you write `href="/blog/"` ???  <-- if this is true, then I should NOT say write a localized URL

Is the above example correct? should I change the wording?

Reply:
That's correct. The URL `/blog` should be reflected in a folder in `src/pages/blog/index.astro`. With `prefix-other-locales`, it's **assumed** that the content _outside_ of a localized folder belongs to the default locale.

> Is the above example correct? should I change the wording?

I think so, we should mention "localized URL" only for those locales that **aren't** the default locale.